README.md

```diff
@@ -775,13 +775,33 @@ parameters:
 
 #### Note about raw block volumes
 
-Applications can use volumes provisioned by PMEM-CSI as 
-[raw block devices](https://kubernetes.io/blog/2019/03/07/raw-block-volume-support-to-beta/).
+Applications can use volumes provisioned by PMEM-CSI as [raw block
+devices](https://kubernetes.io/blog/2019/03/07/raw-block-volume-support-to-beta/). Such
+volumes use the same "fsdax" namespace mode as filesystem volumes
+and therefore are block devices. That mode only supports dax (=
+`mmap(MAP_SYNC)`) through a filesystem. Pages mapped on the raw block
+device go through the Linux page cache. Applications have to format
+and mount the raw block volume themselves if they want dax. The
+advantage then is that they have full control over that part.
+
+Volumes with "devdax" namespace mode are not supported. That mode
```

Comment:
I feel like mentioning here in Blockvolumes section about `devdax` is not appropriate as we have not mentioned `devdax` as supported namespace mode anywhere in our documentation

Reply:
Which is exactly the problem that I am trying to solve here: if we don't mention it anywhere, how does a reader know that it really isn't supported?

This felt like a good place because in some ways it would be more natural to use "devdax" for raw block devices. Some users might come here with that expectation.

Reply:
We have listed supported namespace modes under `Architecture and Operation` section.
> |Namespace modes    |*fsdax, sector* mode<sup>3</sup> namespaces pre-created as pools   |namespace in required mode created directly, no need to pre-create pools   |

If you would like to add a note about why we are not supporting `devdax` that would be the right place in my opinion.

Reply:
Moved.